README.md

```diff
@@ -764,6 +766,51 @@ It becomes then possible to retrieve past values :
 - ```<name[0]>``` uses the first value taken by the key
 - ```<name[1]>``` uses the second element taken by the key
 
+## JSON matchers
+
+The built-in JSON DSL makes it possible to express detailed assertions regarding the values of JSON fields.
+
+If a field is unstable you can also decide to ignore it completely but what if you do not care about the precise value but still want to make sure it has a certain property.
+
+JSON matchers are designed for this use case, they work more or less like placeholders in practice.
+
+```scala
+And assert body.ignoring("city", "realName", "publisher.location").is(
+  """
+  {
+    "name": "<<any-alphanum-string>>",
```

Comment:
The quotes are necessary to have a valid expected JSON to parse to find the matchers.

The derived JsonPath are applied to the actual result; it would work fine since we have decided to use JSON as an input for the matchers.

We could add
```scala
  val anyString = Matcher(
    key = "any-string",
    description = "checks if the field is a String",
    predicate = _.isString
  )
```

it will not be valid if the JsonPath returns a JSON instance different than JString.

For info ```is-present``` was defined such that it would catch anything defined

```scala
  val isPresent = Matcher(
    key = "is-present",
    description = "checks if the field is defined",
    predicate = !_.isNull
  )
```
@peter-gerhard you can write a unit-test if you are not convinced :)